src/main/java/com/rarchives/ripme/ripper/rippers/EroShareRipper.java

```diff
@@ -51,22 +51,34 @@ public void downloadURL(URL url, int index) {
     }
     @Override
     public boolean canRip(URL url) {
-        Pattern p = Pattern.compile("^https?://[w.]*eroshare.com/([a-zA-Z0-9\\-_]+)/?$");
+        Pattern p = Pattern.compile("^https?://eroshae.com/([a-zA-Z0-9\\-_]+)/?$");
```

Comment:
Should probably stick to allowing optional `www` (`which is what the `[w.]*` was accomplishing although `(www[.])?` documents the intent better and would likely be more efficient under the regex engine.